### PR TITLE
Styrka Atmel Custom PCB

### PR DESCRIPTION
**Pull request by nopunin10did**
## Description
Adding firmware for a custom Atmel-based hotswap PCB for the Viktus Styrka.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by nopunin10did** [comment]
Confused as to why lint is failing:
`nopunin10did/styrkatmel: The keymap via should not exist!`

Isn't it fairly standard now for a via keymap to be included alongside the default?

**Comment by keyboard-magpie** [comment]
As of https://github.com/qmk/qmk_firmware/pull/24322 Via keymaps are no longer being held in the main QMK repo and have moved to the VIA userspace (https://github.com/the-via/qmk_userspace_via) instead, where a separate PR will be required.

File: keyboards/nopunin10did/styrkatmel/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Viktus Styrka Atmel Edition
+
+![Styrka](https://i.imgur.com/)
```
Comment (waffle87) [inline keyboards/nopunin10did/styrkatmel/readme.md:3]: Missing link?

File: keyboards/nopunin10did/styrkatmel/readme.md
Comment (waffle87) [inline keyboards/nopunin10did/styrkatmel/readme.md:1]: Please add bootloader instructions per [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md?plain=1).

File: keyboards/nopunin10did/styrkatmel/readme.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The Viktus Styrka is a 65% + left macro block created in a collaboration of OneCreativeMind and BlindAssassin111 (Viktus owner).
+
+This is a custom Atmel ISO Hotswap PCB for the Viktus Styrka designed and maintained by NoPunIn10Did.
+
+* Firmware Maintainer: [nopunin10did](https://github.com/nopunin10did)
+* Hardware Supported: Styrka
+* Hardware Availability: <https://www.Viktus.Design>
```
Comment (waffle87) [inline keyboards/nopunin10did/styrkatmel/readme.md:11]: Could you please add a link to where this PCB can be found.

File: keyboards/nopunin10did/styrkatmel/readme.md (hunk 1 above)
Comment (nopunin10did) [reply keyboards/nopunin10did/styrkatmel/readme.md:3]: Yeah, will fix.

File: keyboards/nopunin10did/styrkatmel/readme.md (hunk 2 above)
Comment (nopunin10did) [reply keyboards/nopunin10did/styrkatmel/readme.md:11]: It's not commercially available at this time. It's a custom project I built.

Comment (waffle87) [reply keyboards/nopunin10did/styrkatmel/readme.md:11]: Ah, fair enough.

**Review by waffle87 — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**